packages/p2p/src/stores/my-ads-store.js

```diff
@@ -519,13 +537,30 @@ export default class MyAdsStore extends BaseStore {
         this.show_edit_ad_form = show_edit_ad_form;
     }
 
+    @action.bound
+    setIsSwitchModalOpen(is_switch_modal_open, app_id) {
```

Comment:
```suggestion
    setIsSwitchModalOpen(is_switch_modal_open, app_id) {
```
`app_id`??? hmmm

Reply:
I will rename the variable to make better sense